app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt

```diff
@@ -80,7 +80,7 @@ class CustomBulletSpanTest {
   }
   private val testStringWithCustomBulletSpan = SpannableString("Text With \nBullet Point").apply {
     this.setSpan(
-      CustomBulletSpan(context),
+      CustomBulletSpan(context = context),
       10,
```

Comment:
please add the named argument here too. 

We usually add named argument where we are passing the arguments and just by reading the argument we cannot understand what is the use of that particular argument. So, here we are passing 10 and 20, which is not helpful to understand what it is, so we need named argument here.

Reply:
but "setSpan" is not a kotlin function,  so named arguments are not allowed in it.

Reply:
ah, this is catch here. Thanks for this update.